testing/conftest.py

```diff
@@ -33,6 +39,51 @@ def pytest_sessionstart():
         install_named_dataset('sct_testing_data', dest_folder=sct_test_path())
 
 
+def pytest_sessionfinish():
+    """Perform actions that must be done after the test session."""
+    # get the newest temporary path created by pytest
+    tmp_paths = glob(os.path.join(tempfile.gettempdir(), "pytest-of-*", "pytest-current"))
+    ctimes = [os.path.getctime(p) for p in tmp_paths]
+    tmp_path = tmp_paths[ctimes.index(max(ctimes))]
+
+    # generate directory summaries for both sct_testing_data and the temporary directory
+    for (folder, fname_out) in [(tmp_path, "pytest-tmp.json"),
+                                (sct_test_path(), "sct_testing_data.json"),
+                                (sct_test_path().replace("testing", "example"), "sct_example_data.json")]:
```

Comment:
I think `sct_test_path()` is subtly the wrong function to use to locate `sct_example_data`, because [it takes into account the `SCT_TESTING_DATA` environment variable](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/d218b9e6d62e46e8d78bd768130a4188554455fd/spinalcordtoolbox/utils/sys.py#L390-L407) (although possibly this is a vestigial feature that we should polish away?). Whereas I think `sct_example_data` is always installed at [the default location](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/d218b9e6d62e46e8d78bd768130a4188554455fd/spinalcordtoolbox/utils/sys.py#L529):
```suggestion
                                (os.path.join(__data_dir__, "sct_example_data"), "sct_example_data.json")]:
```
(This also needs an extra import of `__data_dir__` from `utils.sys` at the top.)